main/src/main/scala/sbt/SlashSyntax.scala

```diff
@@ -0,0 +1,148 @@
+package sbt
+
+import java.io.File
+import sbt.librarymanagement.Configuration
+
+/**
+ * ScopeSyntax implements the slash syntax to scope keys for build.sbt DSL.
```

Comment:
What is "ScopeSyntax"? Either it should read "SlashSyntax" or it needs more explanation.

Reply:
ScopeSyntax was the older name. I'll change this.